<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FruitMaster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;日志信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="countApple">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="countBanana">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="countOrange">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="countMongo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="countPineapple">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushTry">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushTry</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Status</string>
    </property>
    <addaction name="actionMenu_demo"/>
    <addaction name="actionEnd"/>
    <addaction name="actionStep"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Switch Fruit</string>
    </property>
    <addaction name="actionApple"/>
    <addaction name="actionBanana"/>
    <addaction name="actionOrange"/>
    <addaction name="actionMongo"/>
    <addaction name="actionPineApple"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_Run"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHit">
    <property name="title">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuText_Show">
    <property name="title">
     <string>Text Show</string>
    </property>
    <addaction name="actionFruit_Info"/>
    <addaction name="actionLog"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menu"/>
   <addaction name="menuHit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuText_Show"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMenu_demo">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionEnd">
   <property name="text">
    <string>End</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionApple">
   <property name="text">
    <string>Apple</string>
   </property>
  </action>
  <action name="actionBanana">
   <property name="text">
    <string>Banana</string>
   </property>
  </action>
  <action name="actionOrange">
   <property name="text">
    <string>Orange</string>
   </property>
  </action>
  <action name="actionMongo">
   <property name="text">
    <string>Mongo</string>
   </property>
  </action>
  <action name="actionPineApple">
   <property name="text">
    <string>PineApple</string>
   </property>
  </action>
  <action name="actionHow_Run">
   <property name="text">
    <string>How Run</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFruit_Info">
   <property name="text">
    <string>Fruit Info</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
